src/cmd/src/datanode.rs

```diff
@@ -99,15 +98,16 @@ struct StartCommand {
     http_addr: Option<String>,
     #[clap(long)]
     http_timeout: Option<u64>,
+    #[clap(long, default_value = "DATANODE")]
+    env_prefix: String,
 }
 
 impl StartCommand {
     fn load_options(&self, top_level_opts: TopLevelOptions) -> Result<Options> {
-        let mut opts: DatanodeOptions = if let Some(path) = &self.config_file {
-            toml_loader::from_file!(path)?
-        } else {
-            DatanodeOptions::default()
-        };
+        let mut opts: DatanodeOptions = Options::load_layered_options(
+            self.config_file.as_ref().map(|s| s.as_ref()),
+            self.env_prefix.as_ref(),
+        )?;
 
```

Comment:
Looks like it can be simplified with:
```suggestion
        let mut opts: DatanodeOptions =
            Options::load_layered_options(self.config_file.as_deref(), self.env_prefix.as_ref())?;
```

Reply:
TIL